test/unit/object/test_file_request.py

```diff
@@ -0,0 +1,80 @@
+import json
+
+from boxsdk.config import API
+from boxsdk.object.folder import Folder
+from boxsdk.object.file_request import FileRequest
+from boxsdk.object.file_request import StatusState
+
+
+def test_get(test_file_request, mock_box_session):
+    expected_url = f'{API.BASE_API_URL}/file_requests/{test_file_request.object_id}'
+    mock_box_session.get.return_value.json.return_value = {
+        'type': test_file_request.object_type,
+        'id': test_file_request.object_id,
+        'title': 'File Request'
+    }
+    file_request = test_file_request.get()
+    mock_box_session.get.assert_called_once_with(expected_url, headers=None, params=None)
+    assert isinstance(file_request, FileRequest)
+    assert file_request['type'] == file_request.object_type
+    assert file_request['id'] == file_request.object_id
+    assert file_request['title'] == 'File Request'
+
+
+def test_update(test_file_request, mock_box_session):
+    new_title = 'New File Request Title'
+    new_status = StatusState.INACTIVE
+    expected_url = f'{API.BASE_API_URL}/file_requests/{test_file_request.object_id}'
+    mock_box_session.put.return_value.json.return_value = {
+        'type': test_file_request.object_type,
+        'id': test_file_request.object_id,
+        'title': new_title,
+        'status': new_status,
+    }
+    data = {
+        'title': new_title,
+        'status': StatusState.INACTIVE,
+    }
+    file_request = test_file_request.update_info(data=data)
+    mock_box_session.put.assert_called_once_with(expected_url, data=json.dumps(data), headers=None, params=None)
+    assert isinstance(file_request, FileRequest)
+    assert file_request['type'] == test_file_request.object_type
+    assert file_request['id'] == test_file_request.object_id
+    assert file_request['title'] == new_title
+    assert file_request['status'] == StatusState.INACTIVE
+
+
+def test_copy(test_file_request, mock_box_session):
```

Comment:
please test also expires_at field. Make it parametrised to check string and datetime object.